CMakeLists.txt

```
@@ -286,6 +287,11 @@ if (OVERTE_BUILD_CLIENT)
 
   option(USE_SIXENSE "Build Interface with sixense library/plugin" OFF)
   option(USE_NEURON "Build Interface with Neuron library/plugin" OFF)
+  if (OVERTE_USE_SYSTEM_LIBS)
+      set(USE_STEAMCLIENT OFF)
```

```suggestion
      set(USE_STEAMCLIENT OFF)
      message(STATUS "Not using STEAMCLIENT for system library build.")
```
Since we might have the USE_STEAMCLIENT cache variable set (either from previous runs or from someone having it enabled in their IDE), overriding it using a normal variable should probably throw a message. (Usually I would make this a NOTICE, but some IDEs would interpret that as a warning and the Steam client library really isn't that important.)